internal/flavors/vulnerability.go

```diff
@@ -53,9 +54,8 @@ func NewVulnerability(b *beat.Beat, cfg *agentconfig.C) (beat.Beater, error) {
 	// Override trivy's logger
 	scanLog := zap.New(log.Core()).Sugar()
 	dlog.SetLogger(scanLog)
-	flog.SetLogger(scanLog)
 	xlog.SetLogger(scanLog)
-	tlog.Logger = scanLog
+	tlog.SetDefault(slog.New(zapslog.NewHandler(log.Core(), nil)))
```

Comment:
Fix breaking change introduced by zap replacement by slog (https://github.com/aquasecurity/trivy/pull/6466)
I find it weird that trivy uses the [default logger](https://github.com/aquasecurity/trivy/blob/main/pkg/log/logger.go) for everything. But that's why I used SetDefault from github.com/aquasecurity/trivy/pkg/log, which overrides slog default logger